pilot/pkg/kube/inject/testdata/inject/app_probe/hello-probes-with-flag-set-in-annotation.yaml.injected

```diff
@@ -117,6 +117,10 @@ spec:
         - name: ISTIO_METAJSON_LABELS
           value: |
             {"app":"hello","tier":"backend","track":"stable"}
+        - name: ISTIO_META_WORKLOAD_NAME
+          value: hello
+        - name: ISTIO_META_OWNER
```

Comment:
shouldn't this be service account?

Reply:
that would not match the existing contract for the attribute `*.owner.` the naming may be bad, but that is what we've had for a long time now. there is a separate attribute for service account.

Reply:
Added `SERVICE_ACCOUNT` as well, for completeness.